File: algorithms/remove-duplicates-from-sorted-array/javascript/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (nums) {
  let insertIndex = 1;
  for (let i = 1; i < nums.length; i++) {
    if (nums[i] != nums[i - 1]) {
      nums[insertIndex] = nums[i];
      insertIndex++;
    }
  }
  return insertIndex;
};

File: algorithms/remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {
        let mut insertIndex = 1;
        for i in 1..nums.len() {
            if nums[i] != nums[i - 1] {
                nums[insertIndex] = nums[i];
                insertIndex += 1;
            }
        }
        insertIndex as i32
    }
}
